c7n/resources/s3.py

```diff
@@ -390,7 +390,7 @@ def handle_BucketWebsiteConfiguration(self, resource, item_value):
     ('get_bucket_website', 'Website', None, None),
     ('get_bucket_logging', 'Logging', None, 'LoggingEnabled'),
     ('get_bucket_notification_configuration', 'Notification', None, None),
-    ('get_bucket_lifecycle', 'Lifecycle', None, None),
+    ('get_bucket_lifecycle_configuration', 'Lifecycle', None, None),
```

Comment:
@kapilt - I noticed that the S3 augment table is using a [deprecated function](http://boto3.readthedocs.io/en/latest/reference/services/s3.html#S3.Client.get_bucket_lifecycle) `get_bucket_lifecycle`.  I updated it to use the [new function](http://boto3.readthedocs.io/en/latest/reference/services/s3.html#S3.Client.get_bucket_lifecycle_configuration) 

I compared the return schema and the only difference is that the new function includes the `Filter` key, so I don't think there are any compatibility concerns, but I'm calling it out in case this could be a problem.

Reply:
looks fine re delta between the two, thanks for highlighting.